Sources/GDSCommon/UI/UIKit/SharedViews/Components/BulletView.swift

```diff
@@ -2,18 +2,25 @@ import UIKit
 
 /// `BulletView` creates a bulleted list from an array of `String`
 ///  The title is an Optional `String`
+///  The text is of type `[String]`
+///  The titleFont is an Optional `UIFont`
 public final class BulletView: NibView {
     private let title: String?
     private let text: [String]
+    private let titleFont: UIFont
     
     /// Initialiser targets the nib file and requires an array of `String`. From this, the view
     /// constructs a vertical stack as a bulleted list (`bulletStack`)
     /// - Parameters:
     ///   - title: an optional bold formated title to `BulletView`
+    ///   - titleFont: an optional font applied to `title`
```

Comment:
Not really option in the swift sense maybe better to state its has a default value instead

Reply:
Sure, ill add that!